src/main/resources/assets/paths/datasetV3.yaml

```diff
@@ -0,0 +1,39 @@
+post:
+  summary: Creates a set of Datasets from a valid dataset registration JsonSchema object
+  description: Creates a set of Datasets from a valid dataset registration JsonSchema object
+  requestBody:
+    description: | 
+      Dataset registration schema instance and optional file for
+      an alternative data sharing plan. The registration schema is 
+      always required. If a file is also provided, then alternative 
+      data sharing information is required in the registration schema.
+    content:
+      multipart/form-data:
+        schema:
+          type: object
+          properties:
+            dataset:
+              description: Required dataset registration json schema instance
+              type: object
+              $ref: '../schemas/DatasetRegistrationSchemaV1.yaml'
+            file:
+              description: Optional alternative data sharing plan
+              type: string
+              format: binary
```

Comment:
There's not multipart/form-data option in the dropdown.

Reply:
I'm not sure you're looking at the latest version. I see something pretty different. I do see some browser caching issues, but the endpoint looks correct to me:

<img width="1005" alt="Screen Shot 2022-10-03 at 12 28 56 PM" src="https://user-images.githubusercontent.com/116679/193630053-4693136d-ef0d-4c73-8762-ee8f0ef42d8d.png">

Reply:
🤔 curious, I tried emptying my cache (I have had that issue before, but I thought I had fixed it when I emptied the cache). Let me try again..

Reply:
hmmmmm so I cleared the website's cache and the entire browser's cache, and it still has the old information up, which makes no sense because the information here does not relfect what is in swagger. 🤔  very odd

Reply:
OHHH I WAS ON THE WRONG BRANCH, DUOS-1972 instead of DUOS-1972-resource-stub.